app/controllers/smart_proxies_controller.rb

```diff
@@ -6,7 +6,10 @@ class SmartProxiesController < ApplicationController
   before_action :find_status, :only => [:ping, :tftp_server]
 
   def index
-    @smart_proxies = resource_base_search_and_page.includes(:features)
+    # Large content_counts data may cause slow page load, especially with eager load.
+    # So exclude this field since it is not used in index page.
+    needed_columns = SmartProxy.attribute_names.reject { |name| name == "content_counts" }
```

Comment:
Iirc content_counts column on the smart_proxies table is something that comes from katello. It feels weird to accomodate for it from here. Could the exclusion be handled in katello instead?